vllm/platforms/interface.py

```diff
@@ -373,6 +373,16 @@ def use_all_gather(cls) -> bool:
                 or parallel_config.distributed_executor_backend
                 == "external_launcher")
 
+    @classmethod
+    def use_ray(cls) -> bool:
```

Comment:
why do we need a new function in the platform folder, while this is only a check for `parallel_config.distributed_executor_backend == "ray"` ?